### ISSUE TITLE
Main 4 Sections of Content

### ISSUE THREAD

**Issue by pmaji** [pmaji/practical-python-data-viz-guide#4]
Note to self as validation:

For section 3 -- Statistical Visualization in the Wild -- I plan on telling a story surrounding the DTI distributional shift from beginning to end of 2017. This distributional shift can be seen in the QnD image below I build as validation of my hunch when pulling the data. The orange is the late 2017 DTI distro and the blue is the early 2017 DTI distro. 
![Screen Shot 2019-07-04 at 6 45 36 PM](https://user-images.githubusercontent.com/25461122/60699186-d2db2f00-9ec0-11e9-9081-86f6032e00ae.jpg)

**Comment by pmaji** [comment]
Made decent progress on Section 1. Put together a draft of instructions for how others can help with Section 3. Sharing updates with collaborators now. Next steps on this:

1 - Finish notebook content for Section 1. (some to be brought over from DST repo)
2 - Start / finish notebook content for Section 3. 
3 - Start / finish notebook content for Section 4.
4 - Put together slides.

**Comment by pmaji** [comment]
@rtjeannier I have integrated your PR. I gave a look over the entire commit but couldn't preview most of the code because it was too big of a re-write for the GitHub preview. As such, just wanted to clarify:

you only made changes to the code within the section you were working on right? I didn't see other edits when I looked over the full lecture notebook just now, but if you made any, feel free to call them out. 

The only other things that would be nice to get would be:
- code that draws the actual KS test stat LINE on the ecdf charts you made--loves those by the way! 
- p-value chart... but I don't think we'll have time for / need to include that. 
- more commentary for the non-parametric test you did... I don't fully understand some of the output / how to contextualize it

We can go over this tomorrow as I start to make slides! Thanks again for all the help!

PS I'm going to try to annotate your code with more explanatory comments, and also add some more commentary in markdown, so check back and give that a read to be sure I am not mischaracterizing anything you've done!
